methods.go

```diff
@@ -205,12 +207,85 @@ func decodeTorrentsStatusResponse(resp *DelugeResponse) (map[string]*TorrentStat
 	return result, nil
 }
 
+// TorrentError is a tuple of a torrent and an error message, returned by
+// methods that manipulate many torrents at once.
+type TorrentError struct {
+	// ID is the hash of the torrent that experienced an error
+	ID string
+	// Message is a string message
+	Message string
+}
+
+// Error satisfies the error interface
```

Comment:
These kind of GoDoc comments...I would avoid them, I don't really care about making the linter happy about them.